### PR TITLE
Add Microsoft Graph to mtr-device-config-password-include.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9341.

### CODE REVIEW

File: Teams/includes/mtr-device-config-password-include.md [hunk 1]
```diff
@@ -8,9 +8,41 @@ To avoid having to reset the resource account's password and then logging into e
...
+   
+   For details about Update-MgUser, see [Update-MgUser](/powershell/module/microsoft.graph.users/update-mguser).
+    
+   This example sets the password for the account ConferenceRoom01@contoso.com to never expire.
+   
+   ```PowerShell
+   Update-MgUser -UserId <a66eacc0-7c3b-4e02-b58a-39014552d08e> -PasswordPolicies DisablePasswordExpiration
+   ```
+   
+   To get the user id for ConferenceRoom01@contoso.com use the following syntax:
```
Comment (JohanFreelancer9) [inline Teams/includes/mtr-device-config-password-include.md:35]:
```suggestion
   To get the user ID for ConferenceRoom01@contoso.com, use the following syntax:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**